tests/conftest.py

```diff
@@ -316,8 +316,8 @@ def aws(iaas):
 
 @pytest.fixture
 def non_gcp(iaas):
-    if iaas != 'gcp':
-        pytest.skip('test only supported on gcp')
+    if iaas == 'gcp':
```

Comment:
good find. Looks like this was not used anywhere, otherwise it might have enabled/disabled random tests.